financial-services/1.10/modules/ROOT/pages/rpa-setup-guide.adoc

```diff
@@ -0,0 +1,167 @@
+= RPA Setup Guide
+
+This guide provides instructions for implementing the https://anypoint.mulesoft.com/exchange/0b4cad67-8f23-4ffe-a87f-ffd10a1f6873/mulesoft-accelerator-for-financial-services/minor/{fins-version}/pages/Use%20case%207%20-%20Unlock%20AMS%20through%20hyperautomation%20for%20insurance%20brokers/[Unlock AMS Through Hyperautomation for Insurance Brokers] use case, which is part of the MuleSoft Accelerator for Financial Services.
+
+The use case leverages MuleSoft's Robotic Process Automation (RPA) solution to retrieve customer data from your Agency Management System (AMS) and automatically load the data into your CRM. To implement this use case, you must set up RPA on the device that runs the automation process and configure the solution to your specific AMS.
+
+== Before You Begin
+
+* Create an Anypoint Platform account xref:rpa-home::automation-userrolesandpermissions-anypointrpapermissions.adoc[and assign it the `RPA Automations Manager` and `RPA Bots Manager` permissions].
```

Comment:
Can you just create an Anypoint Platform Account? Don't you have to buy the Automation bundle to use this solution?

Reply:
Good question! Are we assuming the customer will have access to create an Anypoint account?